docs/user-guide/how_to_use_stable_preempt_plugin.md

```diff
@@ -0,0 +1,201 @@
+# Stable Preempt Plugin User Guide
+
+## Background
+When we need to enable elastic training or serving, preemptible job's pods can be preempted or back to running repeatedly, if no cool down time set, these pods can be preempted again after they just started for a short time, this may cause service stability dropped.
+So we add "stablepreempt" plugin to ensure preemptible job's pods can run for at least some time set by user.
+
+## Environment setup
+
+### Install volcano
+
+Refer to [Install Guide](../../installer/README.md) to install volcano.
+
+### Update scheduler configmap
+
+After installed, update the scheduler configuration:
+
+```shell
+kubectl edit configmap -n volcano-system volcano-scheduler-configmap
+```
+
+Register `stablepreempt` plugin in configmap while enable `preempt` action
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: volcano-scheduler-configmap
+  namespace: volcano-system
+data:
+  volcano-scheduler.conf: |
+    actions: "enqueue, allocate, preempt, backfill"
+    tiers:
+    - plugins:
+      - name: priority
+      - name: gang
+      - name: conformance
+      - name: stablepreempt
+    - plugins:
+      - name: drf
+      - name: predicates
+      - name: task-topology
+        arguments:
+          task-topology.weight: 10
+      - name: proportion
+      - name: nodeorder
+      - name: binpack
+```
+
+### Running Jobs
+
+Take a simple volcano job as sample.
+
+original job yaml is as below, which has "ps" and "worker" task
+
+```yaml
+apiVersion: batch.volcano.sh/v1alpha1
+kind: Job
+metadata:
+  name: test-job
+spec:
+  minAvailable: 3
+  schedulerName: volcano
+  priorityClassName: high-priority
+  plugins:
+    ssh: []
+    env: []
+    svc: []
+  maxRetry: 5
+  queue: default
+  volumes:
+    - mountPath: "/myinput"
+    - mountPath: "/myoutput"
+      volumeClaimName: "testvolumeclaimname"
+      volumeClaim:
+        accessModes: [ "ReadWriteOnce" ]
+        storageClassName: "my-storage-class"
+        resources:
+          requests:
+            storage: 1Gi
+  tasks:
+    - replicas: 6
+      name: "worker"
+      template:
+        metadata:
+          name: worker
+        spec:
+          containers:
+            - image: nginx
+              imagePullPolicy: IfNotPresent
+              name: nginx
+              resources:
+                requests:
+                  cpu: "1"
+          restartPolicy: OnFailure
+    - replicas: 2
+      name: "ps"
+      template:
+        metadata:
+          name: ps
+        spec:
+          containers:
+            - image: nginx
+              imagePullPolicy: IfNotPresent
+              name: nginx
+              resources:
+                requests:
+                  cpu: "1"
+          restartPolicy: OnFailure
+
+```
+
+#### Edit yaml of vcjob
+
+1. add annotations in volcano job in format below.
+   1. `preemptable` annotation(or label) indicates that job or task is preemptable
```

Comment:
IMO, annotation is better for there exists preemption annotaion `volcano.sh/preemptable: "true"`.